### DISCUSSION TITLE
Is there a way to remap file extensions for syntax highlighting?

### DISCUSSION THREAD

**Discussion by nickvonkaenel** [sxyazi/yazi#1720] [1. Q&A]
I would like to map files ending in ".dat" to be treated as Lua files for syntax highlighting. Is that possible?

**Comment by ze0987** [comment]
Take a look at https://github.com/yazi-rs/plugins/tree/main/mime-ext.yazi

**Comment by nickvonkaenel** [comment]
Thanks!
